poetry.lock

```diff
@@ -4385,13 +4470,13 @@ Jinja2 = ">=2.0"
 
 [[package]]
 name = "synapseclient"
-version = "4.4.0"
+version = "4.4.1"
```

Comment:
Do we need to update synapse python client to 4.4.1 for this feature?

Reply:
For what I wrote yes. It's using this boolean I introduced here: https://github.com/Sage-Bionetworks/synapsePythonClient/compare/v4.4.0...v4.4.1-rc#diff-848a7a4238d8f0a7792848097e227f8e3aa284aeadf15e1fa3d890f17791b89cR280

It also contains a minor improvement to remove 1 HTTP call during login.